src/test/java/com/coinbase/exchange/api/orders/OrderTests.java

```diff
@@ -102,10 +102,29 @@ public void getAllOpenOrders() {
     }
 
     @Test
-    public void getFills() {
-        List<Fill> fills = orderService.getAllFills();
+    public void getFillsByProductId() {
+        List<Fill> fills = orderService.getFillsByProductId("BTC-USD", 100);
         assertTrue(fills.size() >= 0);
     }
+
+    @Test
+    public void createMarketOrderBuyThenGetFillByOrderId() {
+        NewMarketOrderSingle marketOrder = createNewMarketOrder("BTC-USD", "buy", new BigDecimal(0.01));
+        Order order = orderService.createOrder(marketOrder);
+
+        assertTrue(order != null); //make sure we created an order
```

Comment:
assertion seems redundant or possibly we need better tests for `createOrder()` method being called so we don't need to have assertions on preconditions which make the code look relatively untidy/big.

Reply:
Perhaps a mocked response on createOrder would work here?

Reply:
Much nicer XD

Reply:
Thanks! This update has been committed.